### DISCUSSION TITLE
Live Server

### DISCUSSION THREAD

**Discussion by gcohen212** [community/community#118724] [Programming Help]
### Select Topic Area

Question

### Body

I am getting the message that the live server cannot refresh without head and body tabs. But my html code has head and body tabs. What is going wrong?

**Comment by ebndev** [comment]
Hi @Gcohen212, we're happy you're here! 

You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details or providing screenshots might help someone give you a nudge in the right direction.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.
